src/main/java/pl/com/tenderflex/service/UserService.java

```diff
@@ -0,0 +1,17 @@
+package pl.com.tenderflex.service;
+
+import org.springframework.stereotype.Service;
+import lombok.RequiredArgsConstructor;
+import pl.com.tenderflex.dao.UserDao;
+import pl.com.tenderflex.model.User;
+
+@Service
+@RequiredArgsConstructor
+public class UserService {
```

Comment:
IT is good practice to use interfaces 
https://www.linkedin.com/pulse/why-interfaces-recommend-spring-sachin-gorade?trk=read_related_article-card_title#:~:text=Using%20Interfaces%20allows%20your%20classes,without%20changing%20the%20client%20code.

https://davidgiard.com/java-services-and-interfaces-in-a-spring-boot-application